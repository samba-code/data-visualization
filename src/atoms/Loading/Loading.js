import React from "react";
import styled from "styled-components";
import { rem } from "polished";

const StyledLoading = styled.div`
  font-size: ${rem("18px")};
  color: black;
  margin: 20px;
`;

const Loading = () => <StyledLoading>Loading data...</StyledLoading>;

export default Loading;
