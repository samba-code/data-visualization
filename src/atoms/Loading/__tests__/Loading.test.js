import React from "react";
import { render } from "enzyme";

import Loading from "../Loading";

describe("Loading", () => {
  it("should match snapshot", () => {
    const wrapper = render(<Loading />);
    expect(wrapper).toMatchSnapshot();
  });
});
