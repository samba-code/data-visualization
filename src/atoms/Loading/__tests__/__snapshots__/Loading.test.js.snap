// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading should match snapshot 1`] = `
<div
  class="Loading__StyledLoading-accywl-0 dkWplq"
>
  <span
    class="css-0"
  >
    <span
      class="css-129peub"
    />
    <span
      class="css-prnvxp"
    />
    <span
      class="css-8el4s5"
    />
    <span
      class="css-1hf2v34"
    />
    <span
      class="css-tx0a7l"
    />
  </span>
  <p
    class="Loading__LoadingText-accywl-1 cMoPgu"
  >
    Loading data
  </p>
</div>
`;
